packages/shared/notionable-hooks/src/use-summary.tsx

```diff
@@ -757,28 +766,59 @@ export function usePortfolioLiquidationRisk(state: TradeState) {
   } = state;
   const onlyCurrent = !postAccountRisk;
   const intl = useIntl();
+  const theme = useTheme();
   const baseCurrency = useFiat();
+
+  const priorAccountNoRisk =
+    priorAccountRisk === undefined ||
+    (priorAccountRisk?.healthFactor === null &&
+      priorAccountRisk?.liquidationPrice.length === 0);
+
+  const hideArrow = !onlyCurrent && priorAccountNoRisk ? true : false;
+
+  const updatedValue =
+    _h?.updated && _h?.updated > 5
+      ? '5+ / 5.0'
+      : !_h?.updated
+      ? 'No Risk'
+      : ` ${_h?.updated?.toFixed(2)} / 5.0`;
+
+  const textColor =
+    _h?.updated && _h?.updated <= HEALTH_FACTOR_RISK_LEVELS.HIGH_RISK
+      ? colors.red
+      : _h?.updated && _h?.updated <= HEALTH_FACTOR_RISK_LEVELS.MEDIUM_RISK
+      ? colors.orange
+      : _h?.updated && _h?.updated <= HEALTH_FACTOR_RISK_LEVELS.LOW_RISK
+      ? theme.palette.secondary.light
+      : theme.palette.secondary.light;
+
   const healthFactor = {
     ..._h,
     asset: undefined,
-    label: intl.formatMessage({ defaultMessage: 'Health Factor' }),
-    current: _h?.current?.toFixed(2) || 'No Risk',
-    updated: _h?.updated?.toFixed(2) || 'No Risk',
+    label: defineMessages({
+      content: { defaultMessage: 'Health Factor' },
+      toolTipContent: {
+        defaultMessage:
+          'Your health factor shows your risk. A lower health factor means you have more risk. If your health factor drops below 1, you can be liquidated.',
+      },
+    }),
+    current: onlyCurrent ? updatedValue : _h?.current?.toFixed(2) || 'No Risk',
```

Comment:
I don't think this is right, it should use _h?.current instead of `updatedValue`. Maybe you can make a function that does the formatting for the healthFactor and then pass in either h?.current or h?.updated.